Something wrong with package exports

# Bug report

## Packages affected

-   [x] sandpack-client
-   [x] sandpack-react

## Description of the problem
Error from esbuild

```txt
✘ [ERROR] Could not resolve "@codesandbox/sandpack-client"

    ../../node_modules/.pnpm/@codesandbox+sandpack-react@2.0.6_biqbaboplfbrettd7655fr4n2y/node_modules/@codesandbox/sandpack-react/dist/esm/index.js:7:95:
      7 │ ...kClient, extractErrorDetails } from '@codesandbox/sandpack-client';
        ╵                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  Importing the directory "./dist" is forbidden by this package:

    ../../node_modules/.pnpm/@codesandbox+sandpack-react@2.0.6_biqbaboplfbrettd7655fr4n2y/node_modules/@codesandbox/sandpack-client/package.json:21:16:
      21 │       "import": "./dist",
         ╵                 ~~~~~~~~

  The presence of "exports" here makes importing a directory forbidden:

    ../../node_modules/.pnpm/@codesandbox+sandpack-react@2.0.6_biqbaboplfbrettd7655fr4n2y/node_modules/@codesandbox/sandpack-client/package.json:15:2:
      15 │   "exports": {
         ╵   ~~~~~~~~~


```


## What were you doing when the problem occurred?
Trying to build an app using esbuild.

### Your Environment

| Software                | Name/Version |
| ----------------------- | ------------ |
| Sandpack-client version |              |
| Sandpack-react version  |     2.0.6         |
| Browser                 |              |
| Operating System        |              |

--- a/sandpack-client/package.json
+++ b/sandpack-client/package.json
@@ -18,7 +18,7 @@
       "require": "./dist/clients/node/index.js"
     },
     ".": {
-      "import": "./dist",
+      "import": "./dist/index.mjs",
       "require": "./dist/index.js"
     }
   },

